### PR TITLE
[14.0][MIG] mrp_bom_note

### PR DESCRIPTION
**Pull request by lbelorgey**
Migration to 14.0

### CODE REVIEW

File: mrp_bom_note/tests/test_mrp_bom_note.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                "bom_line_ids": [
+                    (0, 0, {"product_id": product_1.id, "product_qty": 4}),
+                    (0, 0, {"product_id": product_2.id, "product_qty": 2}),
+                    (0, 0, {"product_id": product_3.id, "product_qty": 2}),
+                ],
+            }
+        )
+
+        BoM1.write({"notes": "<p>Test</p>"})
+        self.assertEqual(BoM1.notes, "<p>Test</p>")
```
Comment (ivantodorovich) [inline mrp_bom_note/tests/test_mrp_bom_note.py]: This whole test file seems useless, as there's no need to test fields.
There's absolutely no business code in this module 😅 

You can save some penguins by removing it

**Review by ivantodorovich — APPROVED** [review]

File: mrp_bom_note/tests/test_mrp_bom_note.py (hunk 1 above)
Comment (lbelorgey) [reply mrp_bom_note/tests/test_mrp_bom_note.py]: No business code but useful 😉.
Done.

**Review by ivantodorovich — APPROVED** [review]

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Hi. 
There is 2 pot files.
I guess one is useless. Could you remove it ?

Apart of that point.
code review, no test :+1: . Thanks !

**Comment by legalsylvain** [comment]
hi @lbelorgey. Could you take a look on minor remarks ? So we can then merge this usefull module.

thanks !

**Comment by lbelorgey** [comment]
Ok. Done.

**Review by legalsylvain — APPROVED** [review]
Thanks ,!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request closed without merging**